contracts/BurningManager.sol

```diff
@@ -31,23 +34,14 @@ contract BurningManager is Initializable, AccessControlUpgradeable {
 
     // MODIFIERS
 
-    modifier isCharacterOwner(uint256 burnId) {
-        _isCharacterOwner(burnId);
-        _;
-    }
-
-    function _isCharacterOwner(uint256 burnId) internal view {
-        require(hasRole(BURNER_ROLE, msg.sender) || characters.ownerOf(burnId) == msg.sender || garrison.characterOwner(burnId) == msg.sender, 'Not owner');
-    }
-
     modifier isCharactersOwner(uint256[] memory burnIds) {
         _isCharactersOwner(burnIds);
         _;
     }
 
     function _isCharactersOwner(uint256[] memory burnIds) internal view {
         for(uint i = 0; i < burnIds.length; i++) {
-            require(hasRole(BURNER_ROLE, msg.sender) || characters.ownerOf(burnIds[i]) == msg.sender || garrison.characterOwner(burnIds[i]) == msg.sender, 'Not owner');
+            require(characters.ownerOf(burnIds[i]) == msg.sender || garrison.characterOwner(burnIds[i]) == msg.sender, 'Not owner');
```

Comment:
in the future we will be able to add a function to characters and garrison that does this check in bulk, for now it's okay unless we find gas costs unreasonable to begin with